src/shared/sentry/sentryadapter.h

```diff
@@ -61,7 +61,7 @@ class SentryAdapter final : public QObject {
    * or drisk writes.
    * After calling this, any call to sentry is UB.
    */
-  Q_SLOT void onBeforeShutdown();
+  static void onBeforeShutdown();
```

Comment:
Do the comments above this line need be changed because this now called a bit earlier?